[Bug]:  paperwork@2.1.1 non-existent download location - error 404

### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

paperwork@2.1.1

### Expected/Current Behaviour

It should install without any problems, while now it just prints out this:


```
PS C:\Users\ander> scoop install extras/paperwork
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
WARN  To disable this warning, run 'scoop config aria2-warning-enabled false'.
Installing 'paperwork' (2.1.1) [64bit] from extras bucket
Starting download with aria2 ...
Download: 12/05 19:02:32 [ERROR] CUID#8 - Download aborted. URI=https://download.openpaper.work/windows/x86/paperwork-maDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://download.openpaper.work/windows/x86/paperwork-mastDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 87f3d2|ERR |       0B/s|C:/Users/ander/scoop/cache/paperwork#2.1.1#https_download.openpaper.work_windows_x86_paperwork-master-ea1b8b01.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://download.openpaper.work/windows/x86/paperwork-master-ea1b8b01.zip
    referer=https://download.openpaper.work/windows/x86/
    dir=C:\Users\ander\scoop\cache
    out=paperwork#2.1.1#https_download.openpaper.work_windows_x86_paperwork-master-ea1b8b01.zip

ERROR & 'C:\Users\ander\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\ander\scoop\cache\paperwork.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.3 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=14012 --continue --summary-interval=0 --auto-save-interval=1

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=paperwork%402.1.1%3a+download+via+aria2+failed
```

and when i disable Aria2 in config it prints out error 404:


```
PS C:\Users\ander> scoop config aria2-enabled false
'aria2-enabled' has been set to 'false'
PS C:\Users\ander> scoop install extras/paperwork
WARN  Purging previous failed installation of paperwork.
ERROR 'paperwork' isn't installed correctly.
Removing older version (2.1.1).
'paperwork' was uninstalled.
Installing 'paperwork' (2.1.1) [64bit] from extras bucket
The remote server returned an error: (404) Not Found.
URL https://download.openpaper.work/windows/x86/paperwork-master-ea1b8b01.zip is not valid
```

### Steps to Reproduce

```console
PS C:\Users\ander> scoop install extras/paperwork
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
WARN  To disable this warning, run 'scoop config aria2-warning-enabled false'.
Installing 'paperwork' (2.1.1) [64bit] from extras bucket
Starting download with aria2 ...
Download: 12/05 18:57:42 [ERROR] CUID#8 - Download aborted. URI=https://download.openpaper.work/windows/x86/paperwork-maDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://download.openpaper.work/windows/x86/paperwork-mastDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 23e2af|ERR |       0B/s|C:/Users/ander/scoop/cache/paperwork#2.1.1#https_download.openpaper.work_windows_x86_paperwork-master-ea1b8b01.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://download.openpaper.work/windows/x86/paperwork-master-ea1b8b01.zip
    referer=https://download.openpaper.work/windows/x86/
    dir=C:\Users\ander\scoop\cache
    out=paperwork#2.1.1#https_download.openpaper.work_windows_x86_paperwork-master-ea1b8b01.zip

ERROR & 'C:\Users\ander\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\ander\scoop\cache\paperwork.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.3 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=14012 --continue --summary-interval=0 --auto-save-interval=1

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=paperwork%402.1.1%3a+download+via+aria2+failed
PS C:\Users\ander> scoop config aria2-enabled false
'aria2-enabled' has been set to 'false'
PS C:\Users\ander> scoop install extras/paperwork
WARN  Purging previous failed installation of paperwork.
ERROR 'paperwork' isn't installed correctly.
Removing older version (2.1.1).
'paperwork' was uninstalled.
Installing 'paperwork' (2.1.1) [64bit] from extras bucket
The remote server returned an error: (404) Not Found.
URL https://download.openpaper.work/windows/x86/paperwork-master-ea1b8b01.zip is not valid
```


### Possible Solution

The Paperwork devs supposedly changed download location to https://download.openpaper.work/windows/installer/paperwork_stable_installer.exe i assume

### Scoop and Buckets Version

```console
PS C:\Users\ander> scoop --version
Current Scoop version:
af5ffcdd (HEAD -> develop, origin/develop) test(bucket): Skip manifest validation if no manifest changes (#5270)

'ACooper81_scoop-apps' bucket:
f5424674 (HEAD -> master, origin/master, origin/HEAD) FirefoxNightly-Portable: Update to version 109.0a1.20221205093546

'amorphobia_siku' bucket:
813846b (HEAD -> master, origin/master, origin/HEAD) qsv: Update to version 0.76.3

'aoisummer_scoop-bucket' bucket:
ee301cf (HEAD -> master, origin/master, origin/HEAD) update bucket

'cc713_ownscoop' bucket:
b77da87 (HEAD -> master, origin/master, origin/HEAD) clashn: Update to version 2.11

'chawyehsu_dorado' bucket:
303cd769 (HEAD -> master, origin/master, origin/HEAD) firefox-nightly-zh-cn: Update to version 109.0a1.20221204094021

'chenxiangfang_scoop-chenxf' bucket:
942e730 (HEAD -> main, origin/main, origin/HEAD) quiltflower: Update to version 1.9.0

'Cologler_cologler-scoop-bucket' bucket:
21cadc3 (HEAD -> master, origin/master, origin/HEAD) remove tmpwd.json

'ddavness_scoop-roblox' bucket:
67370bf (HEAD -> master, origin/master, origin/HEAD) luau: Update to version 0.555

'deliciousgohan_gohan-bucket' bucket:
c1b6c77 (HEAD -> main, origin/main, origin/HEAD) defold 1.2.192

'DEV-Tools' bucket:
e5419842d (HEAD -> master, origin/master, origin/HEAD) Update paperwork.json

'dodorz_scoop' bucket:
6328c96 (HEAD -> master, tag: PicPick-7.0.1, origin/master, origin/HEAD) PicPick: Update to version 7.0.1

'extras' bucket:
7f2ba648c (HEAD -> master, origin/master, origin/HEAD) wavebox: Update to version 10.108.14.2

'foosel_scoop-bucket' bucket:
24adf8a (HEAD -> main, origin/main, origin/HEAD) freecad-weekly: Update to version 31155

'games' bucket:
6b7d2a20 (HEAD -> master, origin/master, origin/HEAD) yuzu: Update to version 0-1261

'Gladtbam_Bucket' bucket:
5838194 (HEAD -> master, origin/master, origin/HEAD) ventoy: Update to version 1.0.84

'gregwen_grewen-scoop' bucket:
bb72e0e (HEAD -> master, origin/master, origin/HEAD) ocenaudio: Update to version 3.11.17

'hetingde_Meta' bucket:
f00bf1b (HEAD -> master, origin/master, origin/HEAD) add clash-chinese-patch

'hoilc_scoop-lemon' bucket:
4b63cc29 (HEAD -> master, origin/master, origin/HEAD) chirp: Update to version 20221205

'instinctualjealousy_holes' bucket:
ef131ab (HEAD -> master, origin/master, origin/HEAD) AntiMicroX: Removed persist of settings INI, unable to maintain hardlink

'jat001_scoop-ox' bucket:
0d07176 (HEAD -> master, origin/master, origin/HEAD) Update apps list on readme

'java' bucket:
b24e0e5d (HEAD -> master, origin/master, origin/HEAD) temurin8-nightly-jre: Update to version 8.0.362-5.0.202212021814

'KnotUntied_scoop-knotuntied' bucket:
3cc8239 (HEAD -> master, origin/master, origin/HEAD) arrow: Update to version 2.1.0

'LengSword_scoop-yls' bucket:
f854e6e (HEAD -> main, origin/main, origin/HEAD) gopass: Update to version 1.15.0

'main' bucket:
68f3aede6 (HEAD -> master, origin/master, origin/HEAD) gping: Update to version 1.5.0

'MyScoop' bucket:
111541cf4 (HEAD -> master, origin/master, origin/HEAD) Updated xmedia-recode_fixed

'naderi_scoop-bucket' bucket:
16e0297c (HEAD -> master, origin/master, origin/HEAD) vscode-insider: Update to version 1669958312459

'nerd-fonts' bucket:
7f98608 (HEAD -> master, origin/master, origin/HEAD) (chore): fix filter pattern (#214)

'Nirose_node' bucket:
9e45ab0 (HEAD -> main, origin/main, origin/HEAD) unchromium: ^108.0.5359.95

'nonportable' bucket:
6c49cce (HEAD -> master, origin/master, origin/HEAD) spotx-np: Update to version 1.6-20221204-ee520a

'nuke' bucket:
e7ab0fa (HEAD -> master, origin/master, origin/HEAD) sdcc-snapshot-x86_64: Update to version 20221205-13763

'p8rdev_scoop-portableapps' bucket:
6782726 (HEAD -> master, origin/master, origin/HEAD) Update

'php' bucket:
31f073f (HEAD -> master, origin/master, origin/HEAD) php8.0: Update to version 8.0.26

'prezesp_scoop-viewer-bucket' bucket:
7d6f4e3 (HEAD -> master, origin/master, origin/HEAD) [scoop-viewer] Version 0.10.0

'Scoop_Zapps' bucket:
298675a (HEAD -> master, origin/master, origin/HEAD) Auto Update

'su-u_scoop-bucket' bucket:
26178bb (HEAD -> master, origin/master, origin/HEAD) aa

'tomcdj71_scoop-essential-apps' bucket:
fa8d2bb (HEAD -> main, origin/main, origin/HEAD) qbittorrent-enhanced: Update to version 4.5.0.10

'unoun_scoop-bucket' bucket:
f61e814 (HEAD -> main, origin/main, origin/HEAD) Add font-line-seed-jp.json

'versions' bucket:
30ce9447 (HEAD -> master, origin/master, origin/HEAD) ipfilter-nightly: Update to version 1670256141

'vidarkongsli_vidars-scoop-bucket' bucket:
0dc68d4 (HEAD -> master, origin/master, origin/HEAD) Update stupidgit manifest

'WiiDatabase_scoop-bucket' bucket:
c3daae6 (HEAD -> master, origin/master, origin/HEAD) boop: Update to version 2.0.0

'yotiky_scoop-bucket' bucket:
f450152 (HEAD -> master, origin/master, origin/HEAD) Remove json files

'yuhaofe_scoop-drey' bucket:
f8704bd (HEAD -> master, origin/master, origin/HEAD) Add manifest: dijo
```


### Scoop Config

```console
PS C:\Users\ander> scoop config

aria2-warning-enabled : True
alias                 : @{gui=scoop-gui}
aria2-enabled         : True
last_update           : 05/12/2022 18:56:15
aria2.enabled         : True
scoop_branch          : develop
scoop_repo            : https://github.com/ScoopInstaller/Scoop
```


### PowerShell Version

```console
PS C:\Users\ander> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.3.0
PSEdition                      Core
GitCommitId                    7.3.0
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```


### Additional Softwares

_No response_

--- a/bucket/paperwork.json
+++ b/bucket/paperwork.json
@@ -1,12 +1,12 @@
 {
-    "version": "2.1.1",
+    "version": "2.1.2",
     "description": "Personal document manager",
     "homepage": "https://openpaper.work",
     "license": "GPL-3.0-or-later",
     "architecture": {
         "64bit": {
             "url": "https://download.openpaper.work/windows/amd64/paperwork-master-latest.zip",
-            "hash": "1cf4d36a4e14c41e2623fb94d348d6c8fe7d2dfe3f011a0dd85a2eb5c9af7565",
+            "hash": "f35a7f908a7015f7e8ff0f3defa7c7091b40b1e9443c8314c039fdb85d5b77ae"
         }
     },
     "shortcuts": [

--- a/bucket/paperwork.json
+++ b/bucket/paperwork.json
@@ -3,8 +3,12 @@
     "description": "Personal document manager",
     "homepage": "https://openpaper.work",
     "license": "GPL-3.0-or-later",
-    "url": "https://download.openpaper.work/windows/x86/paperwork-master-ea1b8b01.zip",
-    "hash": "559815cdfeb5e62c4ffcbfe41ab3986c3e30d1bc1c63179da9ef80886d0872bd",
+    "architecture": {
+        "64bit": {
+            "url": "https://download.openpaper.work/windows/amd64/paperwork-master-latest.zip",
+            "hash": "1cf4d36a4e14c41e2623fb94d348d6c8fe7d2dfe3f011a0dd85a2eb5c9af7565",
+        }
+    },
     "shortcuts": [
         [
             "paperwork.exe",
@@ -14,11 +18,14 @@
         ]
     ],
     "checkver": {
-        "url": "https://gitlab.gnome.org/api/v4/projects/World%2FOpenPaperwork%2Fpaperwork/repository/tags?order_by=name",
-        "regex": "{[^{}]*\"name\":\"(?<version>[0-9.]+)\"[^{}]*\"commit\":{[^}]*\"short_id\":\"(?<hash>[^\"]+)\"",
-        "replace": "${version}"
+        "url": "https://gitlab.gnome.org/World/OpenPaperwork/paperwork/-/tags?format=atom",
+        "regex": "<title>([\\d.]+)</title>"
     },
     "autoupdate": {
-        "url": "https://download.openpaper.work/windows/x86/paperwork-master-$matchHash.zip"
+        "architecture": {
+            "64bit": {
+                "url": "https://download.openpaper.work/windows/amd64/paperwork-master-latest.zip"
+            }
+        }
     }
 }

